src/cascadia/TerminalApp/TabRowControl.xaml.orig

```diff
@@ -0,0 +1,74 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
+the MIT License. See LICENSE in the project root for license information. -->
+<ContentPresenter
+    x:Class="TerminalApp.TabRowControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:TerminalApp"
+    xmlns:mux="using:Microsoft.UI.Xaml.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <mux:TabView x:Name="TabView"
+                 VerticalAlignment="Bottom"
+                 HorizontalContentAlignment="Stretch"
+                 IsAddTabButtonVisible="false"
+                 TabWidthMode="SizeToContent"
+                 CanReorderTabs="False"
+                 CanDragTabs="False"
+                 AllowDropTabs="False">
+
+        <mux:TabView.TabStripFooter>
+            <mux:SplitButton
+                x:Name="NewTabButton"
+                Click="OnNewTabButtonClick"
+<<<<<<< Updated upstream
+=======
+                Background="Purple"
+>>>>>>> Stashed changes
```

Comment:
A conflict marker!

Reply:
Damn, no idea how this file snuck by, it is removed now.